NewForce-Cohort-4/nutshell-the-lucky-biscuits
# Jp events month

# Description
Filters the events by month and gives each month section a heading with the month name and number of events in that month beside the heading
# Testing Instructions
Add several new events with varying dates to see how it categorizes the events by month
# Checklist:
